docs/generatedcode/python.md

```diff
@@ -6,3 +6,147 @@ type: markdown
+    raise NotImplementedError('Method not implemented!')
+
+  def SuggestFortune(self, request, context):
+    """Replaces the fortune for the given zodiac sign with the provided
+one.
+    errors: invalid zodiac sign
+    """
+    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+    context.set_details('Method not implemented!')
+    raise NotImplementedError('Method not implemented!')
+
+
+def add_FortuneTellerServicer_to_server(servicer, server):
+  rpc_method_handlers = {
+      'TellFortune': grpc.unary_unary_rpc_method_handler(
+          servicer.TellFortune,
+          request_deserializer=fortune_pb2.HoroscopeRequest.FromString,
+          response_serializer=fortune_pb2.HoroscopeResponse.SerializeToString,
+      ),
+      'SuggestFortune': grpc.unary_unary_rpc_method_handler(
+          servicer.SuggestFortune,
+          request_deserializer=fortune_pb2.SuggestionRequest.FromString,
+          response_serializer=fortune_pb2.SuggestionResponse.SerializeToString,
+      ),
+  }
+  generic_handler = grpc.method_handlers_generic_handler(
+      'example.FortuneTeller', rpc_method_handlers)
+  server.add_generic_rpc_handlers((generic_handler,))
+```
+
+## Code Elements
+
+The gRPC code generator starts by importing `grpc` package and the plain `_pb2`
+module, synthesized by `protoc`, which defines non-gRPC-specifc code elements,
+like the classes corresponding to protocol buffers messages and descriptors
+used by reflection.
+
+For each service `~` in the `.proto` file, three primary elements are
+generated:
+
+- [**Stub**](#stub): `~Stub` used by the client to connect to a gRPC service.
+- [**Servicer**](#servicer): `~Servicer` used by the server to implement a
+  gRPC service.
+- [**Registration Function**](#registration-function):
+  `add_~Servicer_to_server` function used to register a servicer with a
+  `grpc.Server` object.
+
+
+### Stub
+
+<a name="stub"></a>The generated `Stub` class is used by the gRPC clients.  It
+will have a constructor that takes a `grpc.Channel` object and initializes the
+stub.  For each method in the service, the initializer adds a corresponding
+attribute to the stub object with the same name.  Depending on the RPC type
+(unary or streaming), the value of attribute will be the output of
+`unary_unary`, `unary_streaming`, or `stream_stream` returned from the
+[channel](http://www.grpc.io/grpc/python/_modules/grpc.html#Channel),
+which will be callable objects of type [UnaryUnaryMultiCallable](http://www.grpc.io/grpc/python/grpc.framework.interfaces.face.html?highlight=unaryunary#grpc.framework.interfaces.face.face.UnaryUnaryMultiCallable), [UnaryStreamMultiCallable](http://www.grpc.io/grpc/python/grpc.framework.interfaces.face.html?highlight=unarystream#grpc.framework.interfaces.face.face.UnaryStreamMultiCallable), or [StreamStreamMultiCallable](http://www.grpc.io/grpc/python/grpc.framework.interfaces.face.html?highlight=streamstream#grpc.framework.interfaces.face.face.StreamStreamMultiCallable).
```

Comment:
The appearance of `framework.interfaces` on this line suggests that you're linking to the wrong classes?